src/period-calculation/generate-fixed-periods/generate-fixed-periods-weekly.ts

```diff
@@ -22,29 +20,34 @@ const getStartingDay = (
     startingDay: number | undefined
 ) => {
     switch (periodType) {
-        case 'WEEKLY':
-        case 'BIWEEKLY':
+        case FIXED_PERIOD_TYPES.WEEKLY:
+        case FIXED_PERIOD_TYPES.BIWEEKLY:
             return startingDay || Days.Monday
-        case 'WEEKLYSAT':
+        case FIXED_PERIOD_TYPES.WEEKLYSAT:
             return Days.Saturday
-        case 'WEEKLYSUN':
+        case FIXED_PERIOD_TYPES.WEEKLYSUN:
             return Days.Sunday
-        case 'WEEKLYTHU':
+        case FIXED_PERIOD_TYPES.WEEKLYTHU:
             return Days.Thursday
-        case 'WEEKLYWED':
+        case FIXED_PERIOD_TYPES.WEEKLYWED:
             return Days.Wednesday
```

Comment:
Also, I think we should try to be consistent and always prefix `period` with `fixed`, since we're going to get into trouble otherwise, if we start having to add relative period logic to this repo.

Reply:
This completely changed, see [my comment further above](https://github.com/dhis2/multi-calendar-dates/pull/12#discussion_r1107934306).